packages/runtime/src/runtime/hooks/useCloseAllDialogs.ts

```diff
@@ -8,9 +8,9 @@ import type {
 export const useCloseAllDialogs: EnsembleActionHook<
   EnsembleActionHookResult
 > = () => {
-  const { closeModal } = useContext(ModalContext) || {};
+  const { closeAllModals } = useContext(ModalContext) || {};
 
-  const closeAllDialogs: () => void = () => closeModal?.();
+  const closeAllDialogs: () => void = () => closeAllModals?.();
```

Comment:
Does this also close modal screens? IIRC, closeAllDialogs does not close modal screens.

Reply:
you are right, lemme fix